firmware/applications/SignBuddy/adc.c

```diff
@@ -0,0 +1,95 @@
+#include "adc.h"
+
+#include "board.h"
+
+#define ADC_PREEMPT_PRIO                     0
+#define ADC_SUB_PRIO                         0
+
+#define VREF                                 ((uint32_t)5000)
+#define ADC_DELAY_CALIB_ENABLE_CPU_CYCLES    (LL_ADC_DELAY_CALIB_ENABLE_ADC_CYCLES * 32)
+#define VAR_CONVERTED_DATA_INIT_VALUE        (__LL_ADC_DIGITAL_SCALE(LL_ADC_RESOLUTION_12B) + 1)
+
+void adc_init(void)
+{
+  LL_ADC_InitTypeDef       ADC_InitStruct       = { 0 };
```

Comment:
nit: yeah this naming sucks lol, I know this is the standard the sdk uses but let's not do it too haha

Reply:
Changing